src/main/java/ch/njol/util/coll/CollectionUtils.java

```diff
@@ -251,6 +251,17 @@ public static boolean containsAnySuperclass(final @Nullable Class<?>[] classes,
 		}
 		return false;
 	}
+
+	/**
+	 * @return whether the given object is an instance of any of the given classes
+	 */
+	public static boolean isAnyInstanceOf(Object object, Class<?>... classes) {
```

Comment:
Is this still used?

Reply:
Yea, it was made for the 'any' variant
![image](https://user-images.githubusercontent.com/29547183/178925858-1c50d3f2-573f-4c8b-9131-a3193915f063.png)

many impls in Skript already have a check like `if (!(e instanceof X || e instanceof Y))`, and I didn't touch them.